ops/Dockerfile

```diff
@@ -39,19 +39,22 @@ LABEL org.opencontainers.image.source=https://github.com/CMSgov/beneficiary-fhir
 # * <https://github.com/solita/docker-systemd>: For Ubuntu, but works!
 # * <https://github.com/moby/moby/issues/28614>: Also some useful info.
 #
-# NOTE: Packages sepecified below should generally be kept up-to-date
+# NOTE: Packages specified below should generally be kept up-to-date
 #       with the dependencies encoded in the BFD platinum image
 #       instructions.
-RUN yum update -y --security && \
+
+RUN rpm --import https://yum.corretto.aws/corretto.key && \
+    curl -L -o /etc/yum.repos.d/corretto.repo https://yum.corretto.aws/corretto.repo && \
```

Comment:
These steps must be replicated in `ops/ansible/playbooks-ccs/roles/platinum/tasks/install-dependencies.yml`